pulpcore/tests/functional/worker_with_coverage.py

```diff
@@ -0,0 +1,16 @@
+import sys
```

Comment:
@bmbouter What do you think about having this located in /tests/ rather than in tasking?  I think we could put it in tasking... we could possibly have different variants for test coverage, cprofiling enabled, and so forth.

Reply:
I think this makes sense. :+1: